index.html

```diff
@@ -90,38 +91,45 @@ <h6 class="article__title">
 									</a>
 								</p>
 							</article>
+						</section>
 
 					</div>
 				</div>
 
 				<div class="sidebar col-lg-4">
```

Comment:
The children in this sidebar need a bit better positioning. Add `display: flex; flex; flex-flow: flex-wrap;` to wrap it's immediate children THEN (see below how to style children):